<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>832</width>
    <height>664</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>801</width>
     <height>601</height>
    </rect>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QTabWidget" name="tabWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>791</width>
       <height>641</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Audio</string>
      </attribute>
      <widget class="QSlider" name="verticalSlider">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>20</y>
         <width>16</width>
         <height>431</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>470</y>
         <width>49</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
      <widget class="QToolButton" name="likeButton">
       <property name="geometry">
        <rect>
         <x>510</x>
         <y>450</y>
         <width>31</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>600</x>
         <y>50</y>
         <width>80</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Playlists</string>
       </property>
      </widget>
      <widget class="QSlider" name="horizontalSlider">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>530</y>
         <width>521</width>
         <height>16</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_play_4">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>500</y>
         <width>80</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_play_5">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>500</y>
         <width>80</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_play_6">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>500</y>
         <width>80</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_play_7">
       <property name="geometry">
        <rect>
         <x>470</x>
         <y>500</y>
         <width>80</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_play">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>500</y>
         <width>80</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_play_3">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>500</y>
         <width>80</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
      <widget class="QToolButton" name="btn_">
       <property name="geometry">
        <rect>
         <x>690</x>
         <y>50</y>
         <width>81</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Add Track</string>
       </property>
      </widget>
      <widget class="QTableView" name="playlistView">
       <property name="geometry">
        <rect>
         <x>600</x>
         <y>80</y>
         <width>171</width>
         <height>451</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="album_label">
       <property name="geometry">
        <rect>
         <x>600</x>
         <y>10</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Current playlist:</string>
       </property>
      </widget>
      <widget class="QLabel" name="album_name_label">
       <property name="geometry">
        <rect>
         <x>700</x>
         <y>10</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_2">
       <property name="geometry">
        <rect>
         <x>460</x>
         <y>340</y>
         <width>80</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
      <widget class="QPushButton" name="editorButton">
       <property name="geometry">
        <rect>
         <x>599</x>
         <y>540</y>
         <width>171</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Editor</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Video</string>
      </attribute>
      <widget class="QToolButton" name="toolButton">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>70</y>
         <width>81</width>
         <height>71</height>
        </rect>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
